src/rust/catpowder/linux/rawsocket/rawsockaddr.rs

```diff
@@ -5,25 +5,14 @@
 // Imports
 //======================================================================================================================
 
-use crate::pal::data_structures::{SockAddr, Socklen};
+use crate::pal::Socklen;
 use ::std::mem;
-
-//======================================================================================================================
-// Constants & Structures
-//======================================================================================================================
+use libc::sockaddr;
 
 #[derive(Clone, Copy)]
-
-/// Raw socket address.
 pub struct RawSocketAddr(libc::sockaddr_ll);
 
-//======================================================================================================================
-// Associate Functions
-//======================================================================================================================
-
-/// Associated functions for raw socket addresses.
```

Comment:
Is there a reason to delete this header?

Reply:
Reverted.